### PR TITLE
docs: add credential signing steps for nym-api

### PR DESCRIPTION
**Pull request by pwnfoo**

### CODE REVIEW

File: documentation/operators/src/nodes/nym-api.md [hunk 1]
```diff
@@ -126,17 +126,69 @@ Initialise your API instance with:
...
+
+#### Generate a new wallet
+
+Begin by generating a new wallet address specifically for your instance to use in credential signing mode. Utilize the `nyxd` command-line tool with the following command:
+
+```
+nyxd keys add signer
+```
+
+It's critical to securely back up the mnemonic phrase generated during this process. This mnemonic is your key to recovering the wallet in the future, so store it in a secure, offline location.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-api.md:142]: I know it should be second nature / shouldn't have to mention this at all, but imo whack this in an `admonish warning` or something just to really emphasize ?

File: documentation/operators/src/nodes/nym-api.md [hunk 2]
```diff
@@ -126,17 +126,69 @@ Initialise your API instance with:
...
+
+
+#### Fund the address
+
+Next, deposit NYM tokens into the newly created wallet address to ensure it can cover transaction fees incurred during the credential signing process. `nym-api` will not operate if the wallet's balance falls below 10 NYM tokens, displaying an error message upon startup.
+
+We recommand beginning with an initial deposit of 100 NYM tokens and monitoring the balance regularly, topping it up as necessary to maintain operational readiness.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-api.md:149]: Do we have an estimate on the token burn rate? E.g. from tests 10 tokens would last X days/weeks?

Comment (pwnfoo) [reply documentation/operators/src/nodes/nym-api.md:149]: This is heavily dependent on the credential signing volume so don't think we can accurately estimate this.

Comment (mfahampshire) [reply documentation/operators/src/nodes/nym-api.md:149]: Gotcha, nw

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**